### PR TITLE
fix: peer dependencies for `webpack serve`

### PR DESCRIPTION
**Pull request by evilebottnawi**
**What kind of change does this PR introduce?**

bugfix

**Did you add tests for your changes?**

No need

**If relevant, did you update the documentation?**

No need

**Summary**

Allow developers to install any webpack-dev-server version (3 or next) + move webpack-cli to peer dependencies, because it is peer dependency

**Does this PR introduce a breaking change?**

Yes, you should add webpack-cli and webpack-dev-server to package.json and install them if you want to use `webpack serve`

**Other information**

No

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Yep, in near future

File: packages/serve/package.json [hunk 1]
```diff
@@ -13,24 +13,18 @@
   "dependencies": {
     "@webpack-cli/utils": "^1.0.1-alpha.5",
     "chalk": "3.0.0",
-    "cross-spawn": "6.0.5",
-    "inquirer": "6.5.1",
-    "webpack-cli": "^4.0.0-beta.8",
-    "webpack-dev-server": "^3.9.0"
+    "cross-spawn": "6.0.5"
```
Comment (evilebottnawi) [inline packages/serve/package.json]: Hm, where we use `cross-spawn` in out code? Maybe it should be dev dependencies?

Comment (ryanclark) [reply packages/serve/package.json]: The only place I see it used in the repo is part of the `package-utils` library

**Comment by evilebottnawi** [comment]
@anshumanv it is `peerDependencies`, developer should install `webpack-dev-server` manually, because we will support webpack-dev-server@3 and webpack-dev-server@4 (next)

**Comment by anshumanv** [comment]
> @anshumanv it is peerDependencies, developer should install webpack-dev-server manually, because we will support webpack-dev-server@3 and webpack-dev-server@4 (next)

Okay makes sense 👍

**Comment by evilebottnawi** [comment]
Thanks

**Pull request merged**